packages/push/src/instruction/float.rs

```diff
@@ -39,6 +39,11 @@ where
 {
     type Error = PushInstructionError;
 
+    #[rustversion::attr(before(1.81), allow(clippy::arithmetic_side_effects))]
+    #[rustversion::attr(
+        since(1.81),
+        expect(clippy::arithmetic_side_effects, reason = "Floats won't overflow")
```

Comment:
Maybe more specific, like "Dividing floats won't overflow"?

Reply:
I don't think any float operations can overflow tho? If you want it to be more specific I can do that.

Reply:
I think you're right. I only thought that being specific about the operation being used might it easier for readers to connect the `expect()` to the operation in question. Not a huge deal, though.